pkg/api/install/install.go

```diff
@@ -34,35 +34,55 @@ import (
 	"k8s.io/kubernetes/pkg/runtime"
 )
 
-// userResources is a group of resources mostly used by a kubectl user
-var userResources = []string{"rc", "svc", "pods", "pvc"}
-
 const importPrefix = "k8s.io/kubernetes/pkg/api"
 
 var accessor = meta.NewAccessor()
 
+// availableVersions lists all known external versions for this group from most preferred to least preferred
+var availableVersions = []unversioned.GroupVersion{v1.SchemeGroupVersion}
```

Comment:
> Sounds like this can be an orthogonal goal. The install package can export the install function, so people can call it after init.

I think so, though I'm not sure that's how its described in the draft doc.  I think that this change helps either way.